cpp/arcticdb/processing/operation_types.hpp

```diff
@@ -520,3 +520,182 @@ bool operator()(T t, const ankerl::unordered_dense::set<U>& u) const {
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::AbsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "ABS");
+    }
+};
+
+template<>
+struct formatter<arcticdb::NegOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::NegOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "-");
+    }
+};
+
+template<>
+struct formatter<arcticdb::IsNullOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::IsNullOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "ISNULL");
+    }
+};
+
+template<>
+struct formatter<arcticdb::NotNullOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::NotNullOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "NOTNULL");
+    }
+};
+
+template<>
+struct formatter<arcticdb::PlusOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::PlusOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "+");
+    }
+};
+
+template<>
+struct formatter<arcticdb::MinusOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::MinusOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "-");
+    }
+};
+
+template<>
+struct formatter<arcticdb::TimesOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::TimesOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "*");
+    }
+};
+
+template<>
+struct formatter<arcticdb::DivideOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::DivideOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "/");
+    }
+};
+
+template<>
+struct formatter<arcticdb::EqualsOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::EqualsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "=");
```

Comment:
==

Reply:
Fixed in [a907db9](https://github.com/man-group/ArcticDB/pull/1557/commits/a907db94c4366371434d048c630f796fcbc7a60e)